jest.config.js

```diff
@@ -11,8 +11,8 @@ module.exports = {
   collectCoverageFrom: [
     '**/lighthouse-core/**/*.js',
     '**/lighthouse-cli/**/*.js',
-    '!**/test/',
-    '!**/scripts/',
+    '!**/test/**/*',
```

Comment:
this wasn't working before now it excludes the tests and script folder. I noticed it because jest hanged on the dbw-large-file.js which was in test/fixtures

Reply:
oh nice! 👍

Reply:
and the build speed is amazing! 10s less on travis 👏 😆